Restructure GitHub Actions CI/CD (#3340)
astronomer/docs: astro/ci-cd-templates/github-actions-private-network.md

---
sidebar_label: Private network templates
title: GitHub Actions template for Private Networks
id: github-actions-private-network
description: Use pre-built Astronomer CI/CD templates to automate deploying Apache Airflow DAGs to Astro on private networks using GitHub Actions.
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> If you do not have access to Astronomer's `deploy-action`, use one of the following [private network templates](#private-network-templates). These templates also work for if you use GitHub Enterprise and can't use the public Astronomer [Deploy Action](https://github.com/astronomer/deploy-action) in the GitHub Marketplace.

Reviewer: ```suggestion
If you don't have access to the Astronomer [deploy action](https://github.com/astronomer/deploy-action) because you can't access the public internet from your GitHub repository, use one of the following private network templates to deploy to Astro. 
```
Not sure how that header made it past the CI checks!
